Kontent.Ai.Management/Modules/ResiliencePolicy/DefaultResiliencePolicyProvider.cs

```diff
@@ -47,8 +38,9 @@ public DefaultResiliencePolicyProvider(int maxRetryAttempts)
 
     private IAsyncPolicy<HttpResponseMessage> WrappedPolicy()
     {
-        var defaultPolicy = Polly.Policy
-            .HandleResult<HttpResponseMessage>(result => StatusCodesToRetry.Contains(result.StatusCode))
+        var defaultPolicy = HttpPolicyExtensions
+            .HandleTransientHttpError()
+            .OrResult(result => result.StatusCode == HttpStatusCode.TooManyRequests)
```

Comment:
So o retry for the other Status codes?

Reply:
The conditions configured to be handled are:
 • Network failures (as System.Net.Http.HttpRequestException)
 • HTTP 5XX status codes (server errors)
 • HTTP 408 status code (request timeout)
 
 Do you miss something?

Reply:
Let me get back with a list.

Reply:
See final resolution down in the comments